packages/deploy-tooling/src/base/deploy.ts

```diff
@@ -64,6 +65,7 @@ export async function getCredentials<T extends BasicAuth | ServiceAuth | undefin
  * @param target - url target configuration
  * @param noPrompt - only if not truthy prompt for anything
  * @param logger - reference to the logger instance
+ * @returns {*}  {(Promise<AbapServiceProvider>)} - ABAP Service Provider
```

Comment:
Why do we add the return type in the doc? Isn't it sufficient to declare it in the signature.
We haven't done that before.

Reply:
the new jsdoc eslint plugin version raises an error in case it is missing. We could disable it if we want, but simpler to keep the defaults IMO.